src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -210,22 +215,35 @@ export default {
 
 			// Check for contentful content
 			const pageEntry = data.contentful?.entries?.items?.[0] ?? null;
-			this.pageData = pageEntry ? processPageContentFlat(pageEntry) : null;
+			this.pageData = pageEntry ? processPageContent(pageEntry) : null;
```

Comment:
Refactor to use `processPageContent` which is the same one we use on ContentfulPage. Maybe remove `processPageContentFlat` in the future